test/fastboot-test.js

```diff
@@ -101,16 +101,15 @@ describe("FastBoot", function() {
       });
   });
 
-  it("can forcefully destroy the app instance using destroyAppInstanceInMs", function() {
+  it("can forcefully destroy the app instance using destroyAppInstanceInMs", function(done) {
```

Comment:
In general I find https://github.com/domenic/chai-as-promised clears all this up nicely.